_sass/_layout.scss

```diff
@@ -243,3 +243,34 @@
 .pagination {
   text-align: center;
 }
+
+// Social Media Biography
+
+.bio-social-media {
+  ul {
```

Comment:
Trying to keep html element targeting to an minimum; can a class be used instead?